BREAKING.md

```diff
@@ -17,6 +17,7 @@ There are a few steps you can take to write a good change note and avoid needing
 - [Remove `IOdspResolvedUrl.sharingLinkToRedeem` and use `IOdspResolvedUrl.shareLinkInfo` instead](#Remove-IOdspResolvedUrl.sharingLinkToRedeem-and-use-IOdspResolvedUrl.shareLinkInfo-instead)
 - [Replace `createCreateNewRequest` function with `createOdspCreateContainerRequest` function](#Replace-createCreateNewRequest-function-with-createOdspCreateContainerRequest-function)
 - [Deprecate IFluidObject and introduce FluidObject](#Deprecate-IFluidObject-and-introduce-FluidObject)
+- [readOnly and readOnlyPermissions removed from container-loader](#readOnly-and-readOnlyPermissions-removed-from-container-loader)
```

Comment:
Please add a corresponding block for this and add more details there similar to the other entries.

Reply:
we're also on 0.53 now so this needs to start a new section